sphinxcontrib/aafig.py

```diff
@@ -100,7 +100,8 @@ def render_aafig_images(app, doctree):
     format_map = app.builder.config.aafig_format
     merge_dict(format_map, DEFAULT_FORMATS)
     if aafigure is None:
-        app.builder.warn('aafigure module not installed, ASCII art images '
+         if app.builder.format != 'man':
```

Comment:
The indentation seems to be inconsistent, it should be 4 spaces instead of 5.

Reply:
You are right! Will fix that